ehr/resources/schemas/dbscripts/sqlserver/ehr-17.21-17.30.sql

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/* Copy of file to deal with EHR version upgrade to 17.21 in onprc17.2Prod branch */
+
+/* ehr-17.20-17.21.sql */
+
+CREATE TABLE ehr.observation_types (
```

Comment:
This file isn't needed anymore. ONPRC is already on 19.1, and bootstrapping servers will run the rollup script.

Reply:
removed